src/commands/build.rs

```diff
@@ -0,0 +1,44 @@
+use std::{
+    env,
+    fs::{self, File},
+    process::Command,
+};
+use wizer::Wizer;
+
+use anyhow::Result;
+
+pub fn handle_build(engine_path: &str, js_path: &str, output_file: &str) -> Result<()> {
+    if env::var("JS_COMPILED").eq(&Ok("1".into())) {
+        env::remove_var("JS_COMPILED");
```

Comment:
Probably it's not a good idea to remove user's env var even if its rarely seen in other use cases.

Reply:
This script is setting the env. var. just before it, so we're just removing what we've set.